src/components/AppRoute.jsx

```diff
@@ -30,7 +29,7 @@ const AppRoute = () => (
         <IndexRoute component={scrollToTopOnMount(Balance)} />
         <Route
           path="details"
-          component={scrollToTopOnMount(AccountDetailsPage)}
+          component={scrollToTopOnMount(BalanceDetailsPage)}
```

Comment:
Nit: Mmh I think AccountDetailsPage was pretty good actually, maybe you renamed it because it can concern a group ? In this case I think AccountOrGroupDetailsPage would be better than BalanceDetailsPage.

Reply:
Yes I struggled to decide how to name it and where to put it actually. If we name it AccountOrGroupDetailsPage, it's also difficult to determine in which duck it should go :(

Reply:
Ahah, I am not really sure it is a good way to decide for the name but fine by me.